src/pkg/auth/auth.go

```diff
@@ -205,7 +209,12 @@ func StartAuthCodeFlow(ctx context.Context, mcpFlow LoginFlow) (AuthCodeFlow, er
 			case 3: // Ctrl-C
 				cancel()
 			case 10, 13: // Enter or Return
-				browser.OpenURL(server.URL)
+				err := browser.OpenURL(server.URL)
+				if err != nil {
+					term.Error("Failed to open browser:", "error", err)
+					cancel()
```

Comment:
If we can't open the browser, the user can still copy-paste the URL themselves, so no need to cancel:
```suggestion
					term.Errorf("failed to open browser: %v", err)
```